include/triton/core/tritonbackend.h

```diff
@@ -1359,6 +1360,54 @@ TRITONBACKEND_ISPEC TRITONSERVER_Error* TRITONBACKEND_ModelInstanceExecute(
     const uint32_t request_count);
 
 
+/// [BETA] Below is an experimental feature and the interface is subject to
```

Comment:
Good question, I will need to think more about it. I was thinking that "beta" may give us flexibility if the current API need to be changed in the future. But the current form doesn't seem to provide such flexibility because it is the same as other APIs in essence. The proper way may be to isolate BETA APIs from this header so the backends can optionally include the BETA header if they decide to use the experimental APIs with the cost of possible backend incompatible issues.

Now we may just drop the mentioning of "beta" and add this API like the others if it looks future-proof / extensible. And try out different practice once we have come up with a rollout strategy.